Installation of keycloak-js fails with npm and yarn

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

adapter/javascript

### Describe the bug

npm:

```bash
$ npm i keycloak-js
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/keycloak-masthead - Not found
npm ERR! 404 
npm ERR! 404  'keycloak-masthead@21.1.0' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lazka/.npm/_logs/2023-04-19T08_47_22_539Z-debug-0.log
```

yarn:

```bash
$ yarn add keycloak-js
yarn add v1.22.19
info No lockfile found.
[1/4] Resolving packages...
Couldn't find any versions for "ui-shared" that matches "21.1.0"
? Please choose a version of "ui-shared" from this list: (Use arrow keys)
Couldn't find any versions for "ui-shared" that matches "21.1.0"
? Please choose a version of "ui-shared" from this list: (Use arrow keys)
error Couldn't find package "keycloak-masthead@21.1.0" required by "@keycloak/keycloak-admin-client@21.1.0" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

### Version

none

### Expected behavior

npm install/yarn add works

### Actual behavior

it fails

### How to Reproduce?

see above

### Anything else?

It works with "21.0.2", but fails with latest aka "21.1.0"

--- a/set-version.sh
+++ b/set-version.sh
@@ -10,11 +10,9 @@ sed -i "s/ENV KEYCLOAK_VERSION .*/ENV KEYCLOAK_VERSION $NEW_VERSION/" quarkus/co
 
 # NPM
 cd js
-npm --workspaces pkg set "dependencies.keycloak-js=$NEW_VERSION"
-npm --workspaces pkg set "dependencies.@keycloak/keycloak-admin-client=$NEW_VERSION"
-npm --workspaces pkg set "dependencies.keycloak-masthead=$NEW_VERSION"
-npm --workspaces pkg set "dependencies.ui-shared=$NEW_VERSION"
-npm version $NEW_VERSION --allow-same-version --no-git-tag-version --workspaces
+npm --workspace=admin-ui --workspace=account-ui --workspace=keycloak-masthead pkg set "dependencies.keycloak-js=$NEW_VERSION"
+npm --workspace=admin-ui pkg set "dependencies.@keycloak/keycloak-admin-client=$NEW_VERSION"
+npm --workspace=keycloak-js --workspace=@keycloak/keycloak-admin-client version $NEW_VERSION --allow-same-version --no-git-tag-version
 cd -
 
 # Documentation

